cmake/README.md

```diff
@@ -234,9 +235,10 @@ make test_spec
 It is a configuration module that is used for the integration of prebuilt oneTBB. It consists of two files (``TBBConfig.cmake`` and ``TBBConfigVersion.cmake``) and can be used via the [find_package](https://cmake.org/cmake/help/latest/command/find_package.html) function.
 
 To use this module in your CMake project:
- 1. Let CMake know where to search for TBBConfig, e.g. specify the location of ``TBBConfig.cmake`` in `TBB_DIR` (for more details about search paths, see [find_package](https://cmake.org/cmake/help/latest/command/find_package.html)).
- 2. Use [find_package](https://cmake.org/cmake/help/latest/command/find_package.html) to find oneTBB.
- 3. Use provided variables and/or imported targets (described below) to work with the found oneTBB.
+ 1. Set TBB_FIND_PACKAGE=ON
+ 2. Let CMake know where to search for TBBConfig, e.g. specify the location of ``TBBConfig.cmake`` in `TBB_DIR` (for more details about search paths, see [find_package](https://cmake.org/cmake/help/latest/command/find_package.html)).
+ 3. Use [find_package](https://cmake.org/cmake/help/latest/command/find_package.html) to find oneTBB.
+ 4. Use provided variables and/or imported targets (described below) to work with the found oneTBB.
```

Comment:
These are the steps for using TBB in different projects using `find_package` functionality. No need to update anything here.

Reply:
Removed the Set TBB_FIND_PACKAGE=ON from instructions.